packages/commonwealth/client/scripts/views/components/CommunityInformationForm/constants.ts

```diff
@@ -37,9 +37,12 @@ const particularChainNodes = (nodeInfo: NodeInfo) => {
   );
 };
 
-const chainIdsWithStakeEnabled = Object.values(
+export const chainIdsWithStakeEnabled = Object.values(
   commonProtocol.factoryContracts,
-).map((c) => c.chainId);
+)
+  // we don't support stake on Blast anymore
```

Comment:
can we add a datetime, like `we don't support stake on Blast anymore as of DD/MM/YY`

Reply:
done